chain-core/Cargo.toml

```diff
@@ -13,7 +13,7 @@ mesalock_sgx = ["sgx_tstd", "secp256k1zkp/sgx"]
 
 [dependencies]
 digest = { version = "0.8", default-features = false}
-tiny-keccak = { version = "1.5.0", default-features = false, features = ["keccak"] }
+tiny-keccak = { version = "2.0", features = ["keccak"] }
```

Comment:
is it `no_std` now? `default-features = false` was there before due to std, if i remember correctly

Reply:
Yes. It is now `no_std`.